tests/tests-tax.php

```diff
@@ -131,6 +131,57 @@ public function test_get_global_tax_rate() {
 		$this->assertEquals( '0.036', edd_get_tax_rate() );
 	}
 
+	public function test_get_tax_rate_post() {
+		$_POST['billing_country'] 	= 'US';
+		$_POST['state'] 			= 'AL';
+		$this->assertEquals( '0.15', edd_get_tax_rate() );
+
+		// Reset to origin
+		unset( $_POST['billing_country'] );
+		unset( $_POST['state'] );
+	}
+
+	/**
+	 * @group tesst
+	 */
+	public function test_get_tax_rate_user_address() {
+
+		// Prep test (fake is_user_logged_in())
+		global $current_user;
+		$current_user 		= new WP_User(1);
+		$user_id 			= get_current_user_id();
```

Comment:
Still getting used to the spaces as indentation :stuck_out_tongue_winking_eye:, I did do it below :smile: